packages/importer/src/importer.ts

```diff
@@ -373,9 +376,10 @@ export class OneSchemaImporterClass extends EventEmitter {
         break
       }
       case "launch-error": {
-        this.emit("launched", {
-          success: false,
-          error: OneSchemaLaunchError.LaunchError,
+        // need to update OS before message will be there
```

Comment:
we should explicitly say `v0.5.0`

Reply:
This is a note to myself. Need to deploy this change: https://github.com/oneschema/oneschema/pull/4529.